testing/idris/APKBUILD

```diff
@@ -0,0 +1,44 @@
+#-*-mode: Shell-script; coding: utf-8;-*-  
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+pkgname=idris
+pkgver=1.0
+pkgrel=0
+pkgdesc="Idris - A Language with Dependent Types"
+url="http://www.idris-lang.org"
+arch="x86_64 armhf"
+license="bsd3"
+depends="libffi gmp ncurses libffi-dev gmp-dev"
```

Comment:
Please remove libffi, gmp and ncurses from depends, these are auto-discovered by abuild, see:

```
>>> idris*: Tracing dependencies...
	libffi
	gmp
	ncurses
	libffi-dev
	gmp-dev
	so:libc.musl-x86_64.so.1
	so:libffi.so.6
	so:libgmp.so.10
	so:libncursesw.so.6
	so:libz.so.1
```

Does Idris really need development files of libffi and gmp?